Command/SshCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+declare(strict_types=1);
+
+namespace Wikimedia\ToolforgeBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+use Wikimedia\ToolforgeBundle\Service\ReplicasClient;
+
+/**
+ * The SshCommand is used to establish an SSH tunnel to the replicas for local environments.
+ * To use, in your application run `php bin/console toolforge:ssh`
+ */
+class SshCommand extends Command
+{
+
+    /** @var string */
+    protected static $defaultName = 'toolforge:ssh';
+
+    /** @var ReplicasClient */
+    protected $client;
+
+    /** @var string */
+    public const LOGIN_URL = 'login.toolforge.org';
+
+    /** @var string Follows the slice and service, i.e. 's1.web' */
+    public const HOST_SUFFIX = '.db.svc.eqiad.wmflabs';
+
+    /**
+     * SshCommand constructor.
+     * @param ReplicasClient $client
+     */
+    public function __construct(ReplicasClient $client)
+    {
+        $this->client = $client;
+        parent::__construct();
+    }
+
+    /**
+     * Configuration for the SshCommand.
+     */
+    protected function configure(): void
+    {
+        $this->setDescription('Create an SSH tunnel to the Toolforge replicas.');
+        $this->setHelp('Note you must already have added your SSH key to the ssh-agent before running this.');
+        $this->addArgument(
+            'username',
+            InputArgument::OPTIONAL,
+            'Your Toolforge UNIX shell username, if different than your local username.'
+        );
+        $this->addOption(
+            'service',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            "Service to use. Either 'web' (default) or 'analytics'",
+            'web'
+        );
+    }
+
+    /**
+     * Create the SSH tunnel and wait until the process is manually killed.
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): void
```

Comment:
Return type should be `int`, but can't be because the parent is void. I think it just has to be left off for now, but the method should still return one of the int codes.

Reply:
Done